react-frontend/src/login/LoginPanel.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import PeerlyTextAndLogo from "shared-components/peerly-logo/PeerlyTextAndLogo";
+import InformativeTextComponent from "login/InformativeTextComponent";
+import { Col, Row } from "core-components/grid/GridComponent";
+import GoogleLoginButton from "login/GoogleLoginButton";
+
+const RowComponent = styled(Row)`
+  height: calc(100% / 3);
+`;
+const FirstCol = styled(Col)`
+  padding-top: 1rem;
+`;
+const Wrapper = styled.div`
+  background-color: ${({ theme }) =>
+    theme === "dark" ? "var(--atomic)" : "var(--white)"};
+`;
+
+const LoginPanel = ({
+  buttonText,
```

Comment:
login panel is going to be rendered from container right? then is it container's job to tell what the button text is going to be? same for `informativeText`, `encouragementThought`, `theme`? whats the point of having this component then? this isn't a shared component as well

Reply:
@mayuriardad informativeText and encouragementThought is change by every organisation that we have discuss so can i set  hard code in LoginPanel or get from props ?

Reply:
@jitendrasbunde https://github.com/joshsoftware/peerly/pull/145/files#diff-ff91d391d53cdce6fb9b96b3e61bf3d3R31 is this not working?
also we are not having different UI per organisation